app/models/event.rb

```diff
@@ -379,26 +379,23 @@ def self.update_crossref(options = {})
     response.results.total
   end
 
-  def self.update_target_doi(options = {})
-    size = (options[:size] || 1000).to_i
-    cursor = (options[:cursor] || [])
-    target_relation_type_id = options[:target_relation_type_id]
-
-    response = Event.query(nil, target_relation_type_id: target_relation_type_id, page: { size: 1, cursor: [] })
-    Rails.logger.info "[Update] #{response.results.total} events with target_relation_type_id #{target_relation_type_id.to_s}."
+  def self.update_target_doi(cursor: nil)
```

Comment:
We could keep the code a bit DRY if we use `loop_through_events`  instead of `update_target_doi`.  And we can remove `update_target_doi` all together.

https://github.com/datacite/lupo/blob/b31a661/app/models/event.rb#L596

Reply:
https://github.com/datacite/lupo/blob/b31a661/app/models/event.rb#L598 should use Event instead of Doi, right?

Reply:
@kjgarza Can you push your suggested changes to the branch. That might be easier.